feature/orchestrator/src/main/java/com/simprints/feature/orchestrator/usecases/response/CreateVerifyResponseUseCase.kt

```diff
@@ -21,7 +23,10 @@ internal class CreateVerifyResponseUseCase @Inject constructor() {
         getFaceMatchResults(projectConfiguration, results),
     ).maxByOrNull { it.confidenceScore }
         ?.let { AppVerifyResponse(it) }
-        ?: AppErrorResponse(AppErrorReason.UNEXPECTED_ERROR)
+        ?: AppErrorResponse(AppErrorReason.UNEXPECTED_ERROR).also {
+            //if subject enrolled with an SDK and the user tries to verify with another SDK
+            Simber.tag(CrashReportTag.MATCHING.name).e("No match results found")
```

Comment:
The message feels a bit too generic, maybe something like "No match results due to SDK mismatch"?

Reply:
@luhmirin-s There are other scenarios where the matching result can be empty, such as a crash in one of the SDKs. This is why I chose a generic log message.